docs_app/src/app/custom-elements/api/api-list.component.html

```diff
@@ -25,7 +25,14 @@ <h2>{{section.title}}</h2>
     <ul class="api-list">
       <ng-container *ngFor="let item of section.items">
         <li *ngIf="item.show" class="api-item">
-          <a [href]="item.path"><span class="symbol {{item.docType}}"></span> {{item.title}}</a>
+          <a [href]="item.path">
+            <span class="symbol {{item.docType}}"></span>
+            <span [ngSwitch]="item.stability">
+              <s *ngSwitchCase="'deprecated'" title="deprecated">{{item.title}}</s>
+              <i *ngSwitchCase="'experimental'" title="experimental">{{item.title}}</i>
+              <span *ngSwitchDefault title="stable">{{item.title}}</span>
+            </span>
```

Comment:
This is a great idea!

I couple of things:

1. To make this more accessible, can we just add the text "(deprecated)" or "(experimental)" next to the name? e.g. `mergeJoin (experimental)` or `merge (deprecated)`. But keep the styling, I like it.
2. Can we do this with a classes instead of markup? It should be a little less overhead, and we're already doing that for the symbol icon.